cli/flox/src/commands/init/node.rs

```diff
@@ -289,19 +457,19 @@ impl Node {
                 (message, result.version.clone())
             },
             (_, Some(NVMRCVersion::Unsure)) => {
-                let result = Self::get_default_node(flox)?;
+                let result = Self::get_default_package("nodejs", flox)?;
```

Comment:
@jennymahmoudi there's a bunch of different messages in the function, but I think you already looked at them. They become the first line in any of the longer prompts I've pasted